erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -67,8 +67,16 @@ def update_opportunity_status(self, opportunity=None):
 			opportunity = self.opportunity
 
 		opp = frappe.get_doc("Opportunity", opportunity)
-		opp.status = None
-		opp.set_status(update=True)
+		quotations_on_opportunity = frappe.db.sql("""
+			SELECT name
+			FROM tabQuotation
+			WHERE `opportunity` = %s AND `status` != "Lost"
+		""",(opportunity))
+
+		if(len(quotations_on_opportunity) <= 1):
+			print("Setting as lost")
+			opp.status = "Lost"
```

Comment:
Why we need to set status here? It should be set based on conditions defined on status_updater.py?